test/localized-data/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "@cap-js/sqlite": "^1"
```

Comment:
Here, we can configure the test through a normal package.json.
The extra file a bit cumbersome though.